### ISSUE TITLE
Having trouble with new folder "pathlib" syntax

### ISSUE THREAD

**Issue by rjwiii** [ecederstrand/exchangelib#260]
Trying to figure out the new "pathlib" format for accessing folders. Having never used pathlib before, I'm having a little trouble. So, if I have the following (deprecated) line:

`my_folder = my_acct.root.get_folder_by_name("My Folder")`

what would be the equivalent using the new "pathlib" syntax?

Thanks.

rjwiii

**Comment by ecederstrand** [comment]
`pathlib` uses the division operator (`/`) to access subfolders. From the README:

```python
some_folder / 'sub_folder' / 'even_deeper' / 'leaf'
```

The equivalent in your case would be:

```python
my_folder = my_acct.root / 'My Folder'
```

to access the direct subfolder of root that is named "My Folder".

**Comment by rjwiii** [comment]
I got this:

Traceback (most recent call last):
  File "C:/Users/M82129/Git/Vendor_Circuit_Notification/testVCN.py", line 134, in <module>
    proc_folder = vcn_acct.root / 'Processed'
TypeError: unsupported operand type(s) for /: 'Root' and 'str'

Python 2.7.10
exchangelib 1.10.2

Thanks,

rjwiii

**Comment by ecederstrand** [comment]
Ah, it didn't work on Python2.7 in v1.10.2. Was fixed in v1.10.3.

**Comment by rjwiii** [comment]
Upgrading worked. Had to dump the tree to find the complete path. Closing.
